src/python/pants/backend/terraform/goals/check.py

```diff
@@ -69,31 +70,43 @@ async def terraform_check(
     if subsystem.skip:
         return CheckResults([], checker_name=request.tool_name)
 
-    initialised_terraforms = await MultiGet(
+    terraform_deployments = await MultiGet(
```

Comment:
Thanks for using call-by-name! In the call-by-name world we spell `MultiGet` as `concurrently`, since there are no underlying `Get`s...